modules/@apostrophecms/storybook/template/public/mocks.js

```diff
@@ -23,10 +23,10 @@
           group: 'AposInputGroup',
           array: 'AposInputArray',
           object: 'AposInputObject',
-          joinByOne: 'AposInputJoinByOne',
-          joinByOneReverse: 'AposInputJoinByOneReverse',
-          joinByArray: 'AposInputJoinByArray',
-          joinByArrayReverse: 'AposInputJoinByArrayReverse',
+          relationshipByOne: 'AposInputRelationshipByOne',
```

Comment:
A few things here that could be updated. 
- relationshipByOne and relationshipByArray should both be `relationship`
- Similar combining for relationshipByOneReverse and relationshipByArrayReverse
- The component for the former is `AposInputRelationship` and `AposInputRelationshipReverse` for the latter.